### DISCUSSION TITLE
Kibana "Application Not Found"

### DISCUSSION THREAD

**Discussion by cnefj384fjfosk** [Security-Onion-Solutions/securityonion#10404] [Unsupported Versions] [answered]
Hi
I am trying to integrate SentinelOne and other applications into Kibana.  
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/116465134/5aecd54b-b808-481c-8a15-1b366d24c9c1)

Got this "Application Not Found" error...
![image](https://github.com/Security-Onion-Solutions/securityonion/assets/116465134/960e5fee-a06d-4ba5-b9b2-0f93f05f4494)

I know that this has been discussed before, but at that time it was related to an earlier version of Elastic. I am on the newest version of Security Onion (2.3.240), which uses Elastic 8.6.2. 
Is this still an issue for anyone else ?

Earlier posts where it has been discussed:
https://github.com/Security-Onion-Solutions/securityonion/discussions/9672
https://www.reddit.com/r/AskNetsec/comments/10lzgdu/security_onion_issue_with_kibana_application_not/

**Comment by dougburks** [comment, marked as answer]
The earlier posts referencing `Application Not Found` are unrelated to what you're seeing. Kibana integrations like SentinelOne are intentionally disabled by default in Security Onion 2.3. We recommend waiting for Security Onion 2.4 (currently in Beta) for better support of these integrations.
